packages/mantine/src/components/prompt/icons/critical.svg

```diff
@@ -0,0 +1,29 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 88 90">
+  <g filter="url(#a)">
```

Comment:
With the iconography I remember we had some problems because the ids (here `a`) would clash with each other since ids are supposed to be unique in the DOM. Perhaps we should add some kind of prefix? Maybe something like `critical-img-a` would be safer.

(same comment applied to other icons)